config/default/core.entity_view_display.node.hs_basic_page.default.yml

```diff
@@ -20,8 +20,8 @@ third_party_settings:
         layout_id: three_column
         layout_settings:
           label: ''
-          section_width: ''
-          region_widths: ''
+          section_width: hs-full-width
+          region_widths: center
```

Comment:
maybe this center value doesnt need to be update 🤔

Reply:
I'm not too concerned about this... it looks like it's the pre-selected value anyways:

![image](https://user-images.githubusercontent.com/1256329/81575173-b88d9f80-9374-11ea-873c-34e8641cda82.png)